src/components/List/List.scss

```diff
@@ -7,4 +7,8 @@
   padding: 0;
   overflow: auto;
   @include scroller();
-}
+
+  &_category--first {
```

Comment:
try use a ":first-child" selector (just for ListTitle if it has a className)

Reply:
Its problematic because its take the first list title in each list title batch (which means it will be relevant to all the list title in the page)